decidim-initiatives/app/permissions/decidim/initiatives/admin/permissions.rb

```diff
@@ -198,10 +198,11 @@ def initiative_user_action?
         end
 
         def allowed_to_send_to_technical_validation?
-          initiative.created? && (
-            !initiative.created_by_individual? ||
-            initiative.enough_committee_members?
-          )
+          initiative.discarded? ||
+            (initiative.created? && (
+              !initiative.created_by_individual? ||
+              initiative.enough_committee_members?
+            ))
```

Comment:
So @oliverbarnes asked a good question--should we `(initiative.discarded? || initiative.created?) && ...` to apply the "created->validating" rules again when recovering a discarded initiative? Any reason not to, since it's probably safer that way?

Reply:
Per: jit.si meeting with Virgile + Armand at OSP, this should be good the way it is; the constraints like `enough_committee_members?` are applicable for the (:created -> :validating) transition, and don't need (or shouldn't) be asked again downstream.